bun/sync-with-meilisearch/src/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { Env } from 'bun';
+import { join, dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+// Throws an error if any of the keys are missing from the object
```

Comment:
Can we use docstring style comments in this file e.g.

```ts
/**
 * Throws an error if any of the keys are missing from the object
 */
```

Reply:
Okay, fixed